### PR TITLE
Feature/camera

### PR DESCRIPTION
**Pull request by andrearastelli**
Camera updated.

Test render:
test_camera

test_camera

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -20,20 +20,28 @@ Color ray_color(const Ray &r, Hitable *world);
 
 int main()
 {
-    Image image("test_diffuse.ppm");
+    Image image("test_camera.ppm");
 
-    auto samples = 8;
+    auto samples = 256;
```
Comment (andrearastelli) [inline src/main.cpp]: ...maybe this is a little too much...

File: src/ray.h [hunk 2]
```diff
@@ -9,6 +9,11 @@
 #include "color.h"
 
 
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__)
+#define M_PI 3.14159265358979323846
+#endif
```
Comment (andrearastelli) [inline src/ray.h]: Because, apparently, on Windows there is no notion of PI.

File: src/main.cpp (hunk 1 above)
Comment (Ale32) [reply src/main.cpp]: Definetly. I'm happy with 8 for now

File: src/camera.h [hunk 3]
```diff
@@ -15,21 +15,45 @@ class Camera
     Vec3 vertical;
 
 public:
-    Camera();
+    Camera(float vfov, float aspect);
```
Comment (Ale32) [inline src/camera.h]: Honestly I prefer horizontal fov, maybe later...

**Pull request merged**